modules/apps/document-library/document-library-video/src/main/java/com/liferay/document/library/video/internal/servlet/filter/DLVideoEmbedFilter.java

```diff
@@ -184,10 +207,89 @@ private String _getFileVersionId(HttpServletRequest httpServletRequest) {
 		return StringPool.BLANK;
 	}
 
+	private Group _getGroup(long companyId, String name)
+		throws PortalException {
+
+		Group group = _groupLocalService.fetchFriendlyURLGroup(
+			companyId, StringPool.SLASH + name);
+
+		if (group != null) {
+			return group;
+		}
+
+		User user = _userLocalService.getUserByScreenName(companyId, name);
+
+		return user.getGroup();
+	}
+
+	private User _getUser(HttpServletRequest httpServletRequest)
+		throws PortalException {
+
+		HttpSession httpSession = httpServletRequest.getSession();
+
+		if (PortalSessionThreadLocal.getHttpSession() == null) {
+			PortalSessionThreadLocal.setHttpSession(httpSession);
+		}
```

Comment:
Why do we need this?

Reply:
Reviewing the code, I think that is not needed.
It's from the original class where I copy the method to obtain the user.

Deleted and pushed

Thanks!!!